corrector.rb

```diff
@@ -0,0 +1,7 @@
+class Corrector
+  def correct_name(name)
+    corrector_name = name.capitalize
+    corrector_name = corrector_name[0..9] if corrector_name.length > 10
+    corrector_name
+  end
```

Comment:
- [OPTIONAL] Your approach is great. But let's remember that, simplicity provides a path for clean, readable and understandable codes. And for this reason, simplicity makes ruby a gem :gem: 

Take a look the this example:

```ruby
def correct_name(name)
    name.capitalize!
    name[0,10]
end
``` 

I bet you love this right? :laughing: You can check out this blog on [how to improve your ruby codes.](improving your Ruby code)

Reply:
Sure bro. 👍 
Thanks for sharing